test/controllers/TransferCompleteControllerSpec.scala

```diff
@@ -38,32 +39,44 @@ class TransferCompleteControllerSpec extends FrontEndTestHelper {
 
   "TransferCompleteController GET" should {
     "render the success page if the export was triggered successfully" in {
-      val client = new GraphQLConfiguration(app.configuration).getClient[gcr.Data, gcr.Variables]()
-      val controller = instantiateTransferCompleteController(getAuthorisedSecurityComponents)
-
-      val consignmentReferenceResponse: gcr.GetConsignment = getConsignmentReferenceResponse
-      val data: client.GraphqlData = client.GraphqlData(Some(gcr.Data(Some(consignmentReferenceResponse))), List())
-      val dataString: String = data.asJson.printWith(Printer(dropNullValues = false, ""))
-      wiremockServer.stubFor(post(urlEqualTo("/graphql"))
-        .willReturn(okJson(dataString)))
+      setConsignmentReferenceResponse()
+      val transferCompleteSubmit = callTransferComplete("consignment")
+      contentAsString(transferCompleteSubmit) must include("Transfer complete")
+      contentAsString(transferCompleteSubmit) must include("TEST-TDR-2021-GB")
```

Comment:
Could you add another `contentAsString(transferCompleteSubmit) must include()` line and then add the "transfer complete" panel text to it?